app/src/main/java/framgia/com/ichat/data/model/Room.java

```diff
@@ -51,5 +51,6 @@ public void setImage(String image) {
 
     public class PrivateRoomKey {
         public static final String PRIVATE_ROOM = "private_room";
+        public static final String EXTRA_ID = "Id";
```

Comment:
e nên để tên value theo tên biến để tránh trường hợp có thể quên và đặt lại giá trị value giống nhau (nhất là trong room)
```suggestion
        public static final String PRIVATE_ROOM = "PRIVATE_ROOM";
        public static final String EXTRA_ID = "EXTRA_ID";
```